Atomic/trust.py

```diff
@@ -478,10 +479,10 @@ def get_gpg_id(self, keys):
         keylist = None
         for key in keys:
             if not os.path.exists(key):
-                # extend when we support parsing inline 'keyData' base64 encoded key
-                return ""
-            else:
-                cmd = ["gpg2", "--with-colons", key]
+                with tempfile.NamedTemporaryFile(delete=False) as tmpkey:
```

Comment:
I think you need to make sure that this file is created on /run, not somewhere a user might be able to attack